packages/core/src/components/widgets/BaseInput.js

```diff
@@ -63,6 +63,7 @@ function BaseInput(props) {
 
   return [
     <input
+      key={inputProps.id}
```

Comment:
What issue did this fix? Could you add that to the PR description?

Reply:
Before adding the key you got key validation errors in e.g. validation_test.js, since it's wrapped in an array:

```
      Custom Form validation
Warning: Each child in a list should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.
    in BaseInput (created by TextWidget)
    in TextWidget
    in Unknown (created by StringField)
    in StringField (created by SchemaField)
    in div (created by WrapIfAdditional)
    in WrapIfAdditional (created by DefaultTemplate)
    in DefaultTemplate (created by SchemaField)
    in SchemaField (created by Form)
    in form (created by Form)
    in Form
```

I updated the PR.